prov/rxm/src/rxm.h

```diff
@@ -61,6 +61,7 @@
 #define RXM_MAJOR_VERSION 1
 #define RXM_MINOR_VERSION 0
 
+#define RXM_CM_DATA_VERSION	1
 #define RXM_OP_VERSION		3
 #define RXM_CTRL_VERSION	3
```

Comment:
This is somewhat unrelated to this PR, but why isn't there just a single version value carried in the first message (i.e. the CM message), rather than having a bunch of different version?

Reply:
The additional versions are for ctrl and op header in the protocol messages. I think they won't be needed anymore as we can check for version compatibility at connection establishment phase. I can do it an a subsequent PR as part of shrinking RxM protocol header (potentially moving away from ofi_proto.h).